src/_common/fireside/post/video/video-model.ts

```diff
@@ -46,6 +47,19 @@ export class FiresidePostVideo extends Model {
 			.sort((a, b) => getManifestPreference(a) - getManifestPreference(b))
 			.map(i => i.img_url);
 	}
+
+	get transcodedVideos(): VideoSourceArray {
+		const videoSources: VideoSourceArray = [];
+		this.media.map(i => {
```

Comment:
change to `.filter.map` then you can return without having the intermediate array.